### ISSUE TITLE
Outlook desktop client - Add in instances freeze after first add in instance is pinned

### ISSUE THREAD

**Issue by jonmoshier** [OfficeDev/office-js#1239]
## Expected Behavior

Add in instances should be responsive to user input.

## Current Behavior

Add in is not responsive to user input.

## Steps to Reproduce, or Live Example

Using Script Lab add in:

1. Open outlook desktop client.
2. Go to inbox and select an email.
3. Open the Script Lab add in.
Addin State: Loads and is responsive to user interaction.

4. Pin add in.
5. Click Reply on email.
Addin State: Add in closed

6. Reopen Script Lab Run Add-in
Add in state: UI is not responsive to user input.

Video of repro: 
[Inbox - jonmoshier_filevine@outlook.com - Outlook (Logging Enabled) 2020-06-24 17-46-23.zip](https://github.com/OfficeDev/office-js/files/4828412/Inbox.-.jonmoshier_filevine%40outlook.com.-.Outlook.Logging.Enabled.2020-06-24.17-46-23.zip)

## Context

Our team is working on our own add in and is seeing this same issue with the Outlook Desktop Client and add in's freezing. I added Script Lab to test this scenario and see that this is an issue for the Script Lab app as well.

If there's a better place to post this bug, please let me know.

## Your Environment
* Platform: PC Desktop, Outlook Desktop Client
* Host: Outlook
* Office version number: Version 2004
* Operating System: Windows 10

**Comment by exextoc** [comment]
Thanks for including the video. I cannot repro so far, following your exact steps.
Could you also include you Windows 10 version? Reason I ask is to confirm whether you are using the Edge WebView control, info here: https://developer.microsoft.com/en-us/office/blogs/microsoft-edge-webview-for-office-add-ins/

**Comment by jonmoshier** [comment]
@exextoc

My system, from System Information:

```
OS Name: Microsoft Windows 10 Pro
Version: 10.0.19041 Build 19041
```

Also, our add in users use Outlook Desktop Client, we don't have control over their platform or version. Thank you!

Thanks for the link, I believe I'm running as a non-subscription user and just saw this bullet in the article.

```
This change does not affect:
* Non-subscription versions of Office for Windows. These will continue to use Internet Explorer 11 as the web viewer.
```

I'll retest this as a subscribed user.

**Comment by jonmoshier** [comment]
I retested this as a subscribed user and it is still an issue. Happy to provide any other info you need.

**Comment by trmccann-fv** [comment]
I wanted to share some additional information.  Using Microsoft Outlook for Microsoft 365 MSO (16.0.12827.20328) 64-bit, I am able to reproduce the issue following the same steps as @jonmoshier.  Further, I am able to reproduce the issue using other add-ins, here's an example of the problem with Salesforce.

[Salesforce-add-in.zip](https://github.com/OfficeDev/office-js/files/4833535/Salesforce-add-in.zip)

**Comment by exextoc** [comment]
You should be on edge webview based on the versions of Outlook. I have also attempted a repro with 16.0.12827 and I don't see the freeze so far. How widespread or roughly how many users are seeing this? Could you try attaching a debugger to see if there are errors reported in the addin? https://docs.microsoft.com/en-us/office/dev/add-ins/testing/debug-add-ins-using-f12-developer-tools-on-windows-10

**Comment by d-franklin** [comment]
I've also experienced the same issue, version Microsoft Outlook for Microsoft 365 MSO (16.0.12827.20328) 64-bit on Widnows 10.

I initially put the issue down to the VM software (KVM/QEMU) I was using, since the issue only manifested for me when running office inside it and I don't have the issue running the same version of office on bare metal or in VMWare.

But based on other peoples experience along with my own, it seems this bug only manifests under certain conditions even when the windows and office version are the same.

I discovered that resizing the outlook window (maximize, restore) after it freezes will unfreeze the add-in, but if you return the original state of the window (restore to maximize then back to restore) it will freeze again (see gif below for an example).

![freeze-resize-example](https://user-images.githubusercontent.com/12947514/85988198-8cb98e00-b9a3-11ea-94de-fe4bda2a4c26.gif)

**Comment by exextoc** [comment]
Unfortunately we haven't been able to repro this in-house. (nor find any known issue that is related to this). Any additional information that can be provided may help us track this down. Does this repro across many of your users / machines? Does it NOT repro in any situations? Can you try upgrading to a later version of Office (if possible) to see if it still repros there? Or a later version of Windows? (Upgrading Windows, will also Upgrade Edge) 

Does this only repro in the pinned scenario? Or does it repro in other cases? (notably maybe in cases where there are multiple taskpanes open across different items?) 

Also the previous question about attaching a debugger to the webview and seeing if there are any errors still applies.

**Comment by arturs-buls** [comment]
We have also encountered this issue while testing. We were able to reproduce with our test app and Salesforce sidebar.

QA was using:
* Office 365 version Outlook (Version 2006 Build 13001.20266), 32-bit
* Windows 10 Version 2004 (Build 19041.329). 
* Edge Version 83.0.478.58. 
Windows/Outlook Updaters do not find any new updates

Pinning is required to reproduce issue. It can not be reproduced with closed/opened, not pinned states. 
Attaching debugger doesn't make any difference. Console log is also empty, if that matters.

Interesting observations: 
* In unresponsive state, if you expand taskpane, the expanded part actually becomes clickable and functional. 
* In unresponsive state, if you open other sidebar it will be not functional too.

**Comment by cbishop-fv** [comment]
Outlook desktop - 16.0.12827.20328
I do have more than one email account setup but using Microsoft 365 for a business account with this test.


I am also able to reproduce this using the salesforce app without even needing an account. The environment picker on the login screen locks up using the original steps. Pinning is required.  

I can also confirm the oddity that resizing the app unfreeze it but resizing it to the original size will resize it.

Also able to reproduce with script lab, boomerang,  ant text (listed in editors choice), XING Pro business (editors choice), Clio, seems fairly wide spread.

**Comment by exextoc** [comment]
We have an in-house repro of this issue. We are currently investigating the root cause.

**Comment by cbishop-fv** [comment]
> We have an in-house repro of this issue. We are currently investigating the root cause.

Thanks! Any updates on this?

**Comment by ifflanb** [comment]
We are experiencing a very similar issue on Windows 10 Build 2004 and Outlook desktop 2009. All web based add-in seem to freeze when replying to emails. They become unfrozen when discarding the email that was created as the reply.
Only possibly related thing that I have also noticed is that when using the office js library in the add-in to change the context based on the newly selected email, the Office.context.mailbox.item is null when you reply. This only happens when replying in Outlook Desktop but not on the web Outlook.

**Comment by ifflanb** [comment]
Is anyone looking into this issue? Is there an update please? thanks

**Comment by exextoc** [comment]
Unfortunately, I cannot give an ETA on the progress being made here, except that it is the appropriate team's backlog.

**Comment by ifflanb** [comment]
Still no update on this? This is still an unresolved issue.

**Comment by Byron-Mo** [comment]
Would also like an update to this as we have a customer case opened that is experiencing the same issue

**Comment by bygate** [comment]
I have a client affected by this issue also. Any updates??

**Comment by exextoc** [comment]
The issue is fixed in all windows builds 21340 and later. Status on backports to older versions of Windows is still being decided.

**Comment by exextoc** [comment]
This issue was fixed, and the fix should be available in Windows build 19043.1266 or later. Note that it is the Windows Build that matters, not the Office build.
